
upper_joint_limit: 0.045 # the upper linear joint limit in meters. This should match the urdf linear joint limit
lower_joint_limit: -0.045 # The lower linear joint limit in meters. This should match the urdf linear joint limit

upper_lin_steps: 230 # the upper linear joint limit for the actual stepper command
lower_lin_steps: 0 # the lower linear joint limit for the actual stepper command

linear_hysteresis: 10 # the smaller the value the more sensative the linear position command generation will be

kick_trigger_threshold: 0.015 # in meters, defines the +/- threshold to trigger a kick

def_rod_xpos: 0.067 # the x value (in world coordinates) of the ideal position for the defensive rod to strike the ball
fwd_rod_xpos: 0.257 # the x value (in world coordinates) of the ideal position for the offensive rod to strike the ball

velocity_modifier: 0.55 # a value from 0 < x <= 1, scales the maximum velocity setting without changing the value in the controller settings.

def_reset_offset: 70 # number of steps to rotate backward to reset after a kick. This is only used for the feedback mode.
fwd_reset_offset: 50 # number of steps to rotate backward to reset after a kick. This is only used for the feedback mode.
full_rotation_offset: 200 # number of steps to complete a full rotation. This is only used for the feedback mode.
